third-party-podspecs/GLog.podspec

```diff
@@ -6,12 +6,34 @@ Pod::Spec.new do |spec|
   spec.summary = 'Google logging module'
   spec.authors = 'Google'
 
+  # Keep in sync with ios-configure-glog.sh
```

Comment:
Not sure you want to do this, but just to put the idea out there -- since this file is just Ruby code, it can read from ios-configure-glog.sh or some other shared file so that the two don't need to manually be kept in sync.